sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -1049,12 +1049,12 @@
           "tags": [],
           "properties": {},
           "conditional": {
-            "show": null,
+            "show": "",
             "when": null,
             "eq": "",
             "json": ""
           },
-          "customConditional": "",
+          "customConditional": "show = data.offeringIntensitySelectedByStudent === 'Part Time'",
```

Comment:
Just a suggestion. We can have a hidden variable(isSelectedOfferingPartTime) which is calculated as `value = data.offeringIntensitySelectedByStudent === 'Part Time'` and use that.  It is just a suggestion not a blocker.

Reply:
Ok. Done!